sonar-maven-plugin/src/main/java/org/sonarsource/scanner/maven/bootstrap/MavenUtils.java

```diff
@@ -94,6 +97,34 @@ static void putAll(Properties src, Map<String, String> dest) {
     }
   }
 
+  /**
+   * Copy all items, except irrelevant encrypted properties, that is,
+   * entries which are encrypted and whose name does not contain "sonar".
+   */
+  static void putRelevant(Properties src, Properties dest) {
+    putRelevant(src, dest::put);
+  }
+
+  /**
+   * See {@link #putRelevant(Properties, Properties)}.
+   */
+  static void putRelevant(Properties src, Map<String, String> dest) {
+    putRelevant(src, dest::put);
+  }
+
+  // Matches strings starting with "{AES}", "{b64}", "{whatever}", etc.
+  private static final Pattern ENCRYPTED_VAR = Pattern.compile("^\\{[a-zA-Z0-9]*\\}.*");
+
+  private static void putRelevant(Properties src, BiConsumer<String, String> dest) {
+    for (String name : src.stringPropertyNames()) {
+      String value = src.getProperty(name);
+      if (ENCRYPTED_VAR.matcher(value).matches() && !name.toLowerCase(Locale.US).contains("sonar")) {
```

Comment:
We could move these conditionals into a dedicated method for readability. WDYT?
This way, we get a higher-level condition and, if new conditions make a property irrelevant, we can add them in the same manner.
```suggestion
      if (isIrrelevantEncryptedProperty(name, value)) {
```

Reply:
Done.